lib/endpoints/class-wp-json-meta-posts-controller.php

```diff
@@ -1,19 +1,24 @@
 <?php
 
-class WP_JSON_Meta_Posts extends WP_JSON_Meta {
+class WP_JSON_Meta_Posts_Controller extends WP_JSON_Meta_Controller {
```

Comment:
Rather than subclassing, let's pass `$type` to `WP_JSON_Meta_Controller`, similar to `WP_JSON_Posts_Controller`